internal/pay/data.go

```diff
@@ -68,13 +68,13 @@ type RefundSummary struct {
 }
 
 type SettlementSummary struct {
-	CaptureSubmitTime string `json:"capture_submit_time"`
-	CapturedDate      string `json:"captured_date"`
-	SettledDate       string `json:"settled_date"`
+	CaptureSubmitTime date.Date `json:"capture_submit_time"`
```

Comment:
![image](https://github.com/user-attachments/assets/ea241190-f480-47bf-8820-81154b3dbc5b)

and this one is `time.Time`